[Bugfix 19570] Tweak usage of bottomLeft/topRight in print docs

MartinKoob opened the pull request:
This is an update to the print.lcdoc file in the dictionary.  I changed references to topLeft to leftTop and edited the descriptions of leftTop and rightBottom parameters for consistency and clarity.

MartinKoob committed:
Create file bugfix-19803.md

bugfix-19803.md added through web interface

MartinKoob committed:
Merge pull request #1 from MartinKoob/patch-2

Create file bugfix-19803.md

MartinKoob committed:
Revert "Create file bugfix-19803.md"

MartinKoob committed:
Merge pull request #2 from MartinKoob/revert-1-patch-2

Revert "Create file bugfix-19803.md"

MartinKoob committed:
Update print.lcdoc to address issue in bug #19570

I have changed the references to topLeft to leftTop as identified.   The leftTop parameter did not have an explanation.  The rightBottom parameter had the explanation for both parameters but was not clear.  I edited the rightBottom parameter explanation to focus on that parameter specifically and added a leftTop description to do the same thing.  Please check to see that the descriptions of the parameters are correct.

MartinKoob committed:
Update print.lcdoc to address bug #19570

There were two entries for the leftTop parameter.  One was blank and one was the last parameter before the description.  I have already added text to the first entry.  I have deleted the second entry.

MartinKoob committed:
Fixing minor mistake an formatting

Ensuring the wording and formatting of the leftTop and rightBottom is consistent.

montegoulding commented:
@MartinKoob I can see why the original author used `leftTop` instead of `topLeft` to try and convey the order of items but as `topLeft` is used throughout the platform I suggest changing the parameters to `topLeft` and `bottomRight`. I think the association is important because it should seem logical to do the following:
```
print this card from the topLeft of group "print" to the bottomRight of print "group"
```

runrevmark commented:
I agree with @montegoulding - for better or for worse, topLeft and bottomRight are the english phrases used to describe the top left point of a rectangle and bottom right point of a rectangle (the order is actually part of the English-language adjective ordering rules - 'left top' sounds wrong, just like we say 'mad old women', rather than 'old mad woman').

MartinKoob committed:
Update print.lcdoc

Changed leftTop to topLeft and rightBottom to bottomRight in response to comments from reviewers.  Added a sentence clarifying the corrordinate order.  I copied that sentence from the mobileAdCreate doc entry.

MartinKoob committed:
added bugfix release note

added release note bugfix-19570.md

MartinKoob commented:
I have revised the print doc using topLeft and bottom right and added a release note.

runrevmark commented:
@MartinKoob : Thanks for this!

@livecode-vulcan review ok f909bf3

livecode-vulcan commented:
💙 review by @runrevmark ok f909bf3c2395ae62cf371a86e2af48d6efc3a79c

livecode-vulcan commented:
😭 test error f909bf3c2395ae62cf371a86e2af48d6efc3a79c

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/258)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/267)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/247)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/190)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/190)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/256)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/261)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/260)
* try-community-x86-linux-debian8: [error](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/265)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/260)
* try-community-x86_64-linux-debian8: [error](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/265)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/284)
* try-community-x86_64-win32: [pending](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/255)

livecodeali commented:
Build failures are irrelevant
